translations/deckbuilding.tex/ua.po

```diff
@@ -375,27 +368,19 @@ msgstr ""
 
 #. type: Plain text
 #: sections/deckbuilding.tex:146
-#, fuzzy
-#| msgid ""
-#| "Spells can be gained by Building the Mage Guild\\index{Mage Guild}.  "
-#| "Building it also enables you to use the Spell Book Token\\index{Spell "
-#| "Book Token} for buying additional Spells.  The Token cannot be used "
-#| "during the \\textbf{same Round} when the Guild is constructed.\\par "
-#| "Spells can be \\pagelink{Trading}{traded} in Alliance and Cooperative "
-#| "Scenarios."
 msgid ""
 "Spells can be gained by Building the Mage Guild\\index{Mage Guild}.  "
 "Building it also enables you to use the Spell Book Token\\index{Spell Book "
 "Token} for buying additional Spells.  The Token cannot be used during the "
 "\\textbf{same Round} when the Guild is constructed. Spells can also be "
 "\\pagelink{Trading}{traded} in Alliance and Cooperative Scenarios."
 msgstr ""
-"Заклинання можна отримати, Збудувавши Гільдію Магів\\index{Гільдія Магів}. "
+"Заклинання можна отримати, Збудувавши Гільдію Магів\\index{Гільдія Магів}.  "
 "Її створення також надає можливість використовувати Жетон Книги "
-"Заклинань\\index{Жетон Книги Заклинань} для купівлі додаткових Заклинань. "
+"Заклинань\\index{Жетон Книги Заклинань} для купівлі додаткових Заклинань.  "
 "Жетон не можна використовувати під час \\textbf{того ж Раунду} в якому була "
-"побудована Гільдія.\\par Заклинаннями можна \\pagelink{Trading}"
-"{обмінюватися} в Сценаріях з режимом Альянс та Кооператив."
+"побудована Гільдія.Заклинаннями можна \\pagelink{Trading}{обмінюватися} в "
```

Comment:
Is the lack of space after the dot intended?

Reply:
Nope